apps/ucp-ui/src/Institutions/utils.ts

```diff
@@ -4,3 +4,7 @@ export const aggregatorIntegrationsSortByName = (
   a: AggregatorIntegration,
   b: AggregatorIntegration,
 ) => a.aggregator.displayName.localeCompare(b.aggregator.displayName);
+
+export const isEmptyStr = (str: string | undefined) => {
```

Comment:
Why don't we store null instead of an empty string?

Reply:
I couldn't figure out a very clean way to do it. Let me try something different and let me know if you think it's better.